data-gathering/bot.py

```diff
@@ -14,48 +15,80 @@
 from datetime import datetime
 from query import store_articles, fetch_wikidata_info, get_id_and_subjects_and_grade
 
-# Open and read the JSON configuration file and access the configuration data as a dictionary
-with open("wikipedia_config.json", "r") as config_file:
-   wikipedia_config = json.load(config_file)
-
 
 # Accept 
 if len(sys.argv) > 2:
-   WIKIPEDIA_LANGUAGE  = sys.argv[1]
+   WIKIDATA_COUNTRY_ID  = sys.argv[1]
    CODE = sys.argv[2]
 else:
    print("Please provide the wikipedia language and code as a command-line argument (e.g., 'bot.py en 117' for Ghana's curriculum in English wikipedia).")
    sys.exit(1)
 
+# Open and read the JSON configuration file and access the configuration data as a dictionary
+with open("wikipedia_config.json", "r") as config_file:
+   wikipedia_config = json.load(config_file)
+
+if WIKIDATA_COUNTRY_ID in wikipedia_config:
+   language_config = wikipedia_config[WIKIDATA_COUNTRY_ID]
+else:
+   print(f"Configuration not found for language '{WIKIDATA_COUNTRY_ID}'.") # Handle this case appropriately (e.g, exit the script).
+
+
+id_wikidata = 1,
```

Comment:
In my opinion these variables should be kept at the config file. My argument for that is that, if I should be able to choose whether to fetch the dimension or not, or whether to fetch the first edit or not, I should be able to do so by changing just the config file, while keeping the code safe and untouched.

However, this is open for discussion and we'll not be touching these paremeters in the near future anyway. We can leave them as is and we can re-discuss later.

Reply:
Alrighty, later it is.